apps/zotonic_mod_menu/src/mod_menu.erl

```diff
@@ -447,16 +461,18 @@ manage_schema(install, Context) ->
     #datamodel{
         categories = [
               {menu, categorization,
-               [{title, <<"Page menu">>}]
+                    #{
+                        <<"title">> => <<"Page menu">>
+                    }
               }
         ],
         resources = [
             {main_menu,
                 menu,
-                [
-                    {title, <<"Main menu">>},
-                    {menu, default_menu(Context)}
-                ]
+                #{
+                    <<"title">> => <<"Main menu">>,
```

Comment:
Same capitalisation.

Reply:
Indeed, I see that the MS also uses capitals for both words.

https://www.microsoft.com/en-us/language/Search?&searchTerm=main%20menu&langID=155&Source=true&productid=undefined